src/FGFDMExec.cpp

```diff
@@ -691,7 +691,11 @@ void FGFDMExec::ResetToInitialConditions(int mode)
 {
   if (Constructing) return;
 
-  if (mode == 1) Output->SetStartNewOutput();
+  // mode flags
+  const int START_NEW_OUTPUT    = 0x1;
+  const int DONT_EXECUTE_RUN_IC = 0x2;
```

Comment:
```suggestion
```
May be we should take the opportunity of your PR to locate **and document** these constants in the header `FGFDMExec.h` ? Currently, only the most motivated users could unearth the existence of these constants and their role.